R/geoms.R

```diff
@@ -8,30 +8,62 @@
 #' @export
 #' @import proto
 #' @example inst/examples/breakpoints.R
```

Comment:
I don't think we need `proto` now. As a sidenote, do we use `proto` somewhere else in the code too? Otherwise we may edit the DESCRIPTION file.